diff --git a/skypeweb/libskypeweb.h b/skypeweb/libskypeweb.h
--- a/skypeweb/libskypeweb.h
+++ b/skypeweb/libskypeweb.h
@@ -175,15 +175,7 @@
 	#define purple_hash_destroy purple_cipher_context_destroy
 	#define purple_xfer_set_protocol_data(xfer, proto_data) ((xfer)->data = (proto_data))
 	#define purple_xfer_get_protocol_data(xfer) ((xfer)->data)
-#if !PURPLE_VERSION_CHECK(2, 11, 0) && !FEDORA
-static inline gboolean
-purple_xfer_write_file(PurpleXfer *xfer, const guchar *buffer, gsize size) {
-	PurpleXferUiOps *ui_ops = purple_xfer_get_ui_ops(xfer);
-	purple_xfer_set_bytes_sent(xfer, purple_xfer_get_bytes_sent(xfer) + 
-		(ui_ops && ui_ops->ui_write ? ui_ops->ui_write(xfer, buffer, size) : fwrite(buffer, 1, size, xfer->dest_fp)));
-	return TRUE;
-}
-#endif
+
 	#define PURPLE_CMD_FLAG_PROTOCOL_ONLY PURPLE_CMD_FLAG_PRPL_ONLY
 	#define PURPLE_CMD_P_PLUGIN PURPLE_CMD_P_PRPL
 	
